### DISCUSSION TITLE
Local Sponsorblock Storage for Playlists

### DISCUSSION THREAD

**Discussion by IndigoSamurott** [ReVanced/revanced-discussions#542] [Question] [answered]
As sponsorblock is server-based, a lack of internet or a server outage .can prevent segments from showing up on videos. Therefore, I think a patch with the ability to store segments offline for videos saved to playlists (possibly opt-in for each list), and refresh them whenever the video is played with the sponsorblock server reachable, would be very useful.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
A simpler solution is to change your SB settings to use a SB API server that has fall over backup support, such as 
https://github.com/hymccord/sb-status

**Reply by IndigoSamurott** [reply]
Thank you for pointing this out. To be honest, having a limited Sponsorblock cache when the _user_ loses connection is the issue I was really thinking about, but I can see why it might not be the most pressing idea.
